doc/manual/manual.tex

```diff
@@ -6795,8 +6795,8 @@ \subsubsection{3D convection with an Earth-like initial condition}
 \end{figure}
 
 
-\marginpar{Use cookbooks/periodic-box.prm}
-\marginpar{Finish the GPlates section}
+%\marginpar{Use cookbooks/periodic-box.prm}
+%\marginpar{Finish the GPlates section}
```

Comment:
I have actually no idea if these lines are still relevant (one is yours, one is Rene's), I just didn't like the point that there are these TODOs in the manual (and now that I am looking, I found a couple of marginpars like these ones). We could put them in a different place (for example as issues on github), or maybe there's a better way, like opening a project on github and than adding all the manual fixes we want to do as notes, but I don't think we should have marginpars in the manual (because that's a also not the place where I would look if I had time for half an hour of free time and wanted to fix something in ASPECT, and also because we will forget to remove them for the releases).